lib/actions/ui.js

```diff
@@ -166,3 +171,32 @@ export function windowMove () {
     });
   };
 }
+
+export function openFile (path) {
+  return (dispatch, getState) => {
+    dispatch({
+      type: UI_OPEN_FILE,
+      effect () {
+        stat(path, (err, stats) => {
+          if (!err) {
```

Comment:
Maybe this error should be handled somehow too? Not sure how, but a console.log or notification is probably a good start.